_layouts/default.html

```diff
@@ -6,70 +6,101 @@
+    <link rel="dns-prefetch" href="//photos3.meetupstatic.com">
+    <link rel="dns-prefetch" href="//photos4.meetupstatic.com">
+    <link rel="dns-prefetch" href="//api.meetup.com">
+    <link rel="dns-prefetch" href="//maps.googleapis.com">
+    <link rel="dns-prefetch" href="//maps.gstatic.com">
 </head>
 <body>
+    <header>
+        <a href="/" class="logo">
+            <img src="/img/JSOxfordLogo.png" alt="JS Oxford"/>
+        </a>
+    </header>
+
     <div class="container">
         <div class="row">
-            <div class="span3 main-nav">
-                <h1>
-                    <a href="/">
-                        <span class="logo">
-                            <img src="/assets/js.png" width="370"  height="370" alt="JS"/>
-                        </span>
-                        Oxford
-                    </a>
-                </h1>
-                <p>
-                    monthly get together near Cowley Road
-                </p>
-                <hr />
-                <p>
-                    {% include mailchimp.html %}
-                </p>
-                <hr />
-                <dl class="links">
-                    <dt>github</dt>
-                    <dd><a href="https://github.com/jsoxford">jsoxford</a></dd>
-                    <dt>twitter</dt>
-                    <dd><a href="https://twitter.com/intent/user?screen_name=jsoxford">@jsoxford</a></dd>
-                    <dt>IRC</dt>
-                    <dd>#jsoxford on irc.freenode.net
-                        <a href="/irc">log</a><br />
-                        <small><a target="_blank" href="http://webchat.freenode.net/?channels=jsoxford">[web&nbsp;client]</a></small></dd>
-                </dl>
-                <hr />
-                <p>
-                    <strong>
-                        like to talk at JSOxford?
-                        <a href="https://docs.google.com/a/whiteoctober.co.uk/spreadsheet/viewform?formkey=dHE0aldWMEFvUGY0ZXprTUlMODBXZkE6MQ">Propose a talk</a>
-                    </strong>
-                </p>
-                <hr />
+            <div class="col-md-4 pull-right sidebar">
+                <h3>events</h3>
+                <div><a href="http://www.meetup.com/JSOxford/"><img src="/img/meetup-logo.png"/></a></div>
+                <h3>github</h3>
+                <div><a href="https://github.com/jsoxford">jsoxford</a></div>
+                <h3>twitter</h3>
+                <div><a href="https://twitter.com/intent/user?screen_name=jsoxford">@jsoxford</a></div>
+                <h3>chat</h3>
+                <div><a href="https://gitter.im/jsoxford/jsoxford.github.com">gitter.im/jsoxford/jsoxford.github.com</a></div>
+                <h3>Want to talk at JSOxford?</h3>
+                <div class="form-group">
+                    <a class="btn btn-primary btn-lg" href="https://docs.google.com/a/whiteoctober.co.uk/spreadsheet/viewform?formkey=dHE0aldWMEFvUGY0ZXprTUlMODBXZkE6MQ">Propose a talk</a>
+                </div>
```

Comment:
Would you mind removing this link?  Been meaning to do it for ages - it's never used, and want to start encouraging people to use github issues.